src/test/java/spoon/test/template/TemplateTest.java

```diff
@@ -442,6 +439,25 @@ public void testCheckBoundTemplate() throws Exception {
 		assertEquals(injectedCode, method.getBody().getStatement(0));
 	}
 
+	@Test
+	public void testAbstractTemplateIsWellFormed() {
+		// contract: IsWellFormed method of AbstractTemplate class returns false if there are zero template parameter
+
+		// arrange
+		AbstractTemplate abstractTemplate = new AbstractTemplate() {
```

Comment:
When naming variables, try to describe their intent instead of their static type. For example, this is more informative:

```suggestion
		AbstractTemplate zeroParameterTemplate = new AbstractTemplate() {
```